frontend/src/components/applyForm/widgets/CheckboxWidget.tsx

```diff
@@ -64,6 +65,9 @@ function CheckboxWidget<
 
   return (
     <FormGroup error={error} key={`form-group__checkbox--${id}`}>
+      {error && (
+        <FieldErrors fieldName={id} rawErrors={rawErrors as string[]} />
```

Comment:
The errors are strings for the widgets, but that was expanded to keep the error object for the multiField.

It would be simpler to just keep the same structure which we could do by updating `getWarningsForField`.